spec/features/observer_spec.rb

```diff
@@ -77,6 +77,22 @@
     expect(submit_button).to_not be_disabled
   end
 
+  it "observer can delete themselves as observer" do
```

Comment:
should we have a test somewhere that an observer cannot delete another observer? maybe a controller-level spec?